redis.conf

```diff
@@ -236,11 +237,13 @@ daemonize no
 #   supervised upstart - signal upstart by putting Redis into SIGSTOP mode
 #                        requires "expect stop" in your upstart job config
 #   supervised systemd - signal systemd by writing READY=1 to $NOTIFY_SOCKET
+#                        on startup, and updating Redis status on a regular
+#                        basis.
 #   supervised auto    - detect upstart or systemd method based on
 #                        UPSTART_JOB or NOTIFY_SOCKET environment variables
 # Note: these supervision methods only signal "process is ready."
 #       They do not enable continuous pings back to your supervisor.
-supervised no
+supervised auto
```

Comment:
that makes the default of redis different than the default of the redis.conf.
the convention used to be that they're the same.
so, let's consider making it the default in redis, and if not, the comment here should probably be improved.
either way, this change the default behavior (of someone using this conf file), so it should maybe be shipped in a major minor version?

Reply:
I'm OK with keeping the default `no` for now but just include a commented out `supervised auto` to make it more obvious it's needed running under systemd.

Reply:
ok, so make that change, and remove the major-decision label and we can merge it.

Reply:
Still a major-decision due to `INFO` field.